services/indexes/models/avax.go

```diff
@@ -124,3 +127,18 @@ type OutputList struct {
 	ListMetadata
 	Outputs []*Output `json:"outputs"`
 }
+
+type CvmOutput struct {
```

Comment:
What is the reason for creating a separate Output struct? At least c chain import inputs and c chain export outputs are the same type of Outpoint in the protocol as any other x or p chain  outpoint.